clarifai/rest/client.py

```diff
@@ -1312,8 +1312,8 @@ def get(self, model_name=None, model_id=None, model_type=None):
         if len(res) > 0:
           # exclude embed and cluster model when it's not explicitly searched for
           if model_type is None:
-            res = list(filter(lambda one: (one.output_info['type'] != u'embed') & (
-              one.output_info['type'] != u'cluster'), res))
+            res = list(filter(lambda one: (one.output_info.get('type') != u'embed') & (
+              one.output_info.get('type') != u'cluster'), res))
```

Comment:
I remember this being similar to one tiktok issue where payload structure change, should we throw a specific error that tells us something has change when this prop does not exist in response?

Reply:
i think it is ok to have as is. if you look at the logic it is just to remove from results if explicitly stated.